### DISCUSSION TITLE
How I can return attributes data human friendly for REST API

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#68] [Q&A] [answered]
As I see product attributes by default return with attribute class name. How I can return attributes human friendly for my REST API?

**Comment by alecritson** [comment, marked as answer]
What are you looking to get back? If you want to return the value you can just call:

```php
$product->translateAttribute('name')
```

If you're using [Eloquent Resources](https://laravel.com/docs/8.x/eloquent-resources) then it's easy to map the data exactly how you need it.
